app/controllers/surveys_controller.rb

```diff
@@ -0,0 +1,51 @@
+class SurveysController < ApplicationController
+  
+  def index
+    @surveys = Survey.all
+  end
+
+  def new
+    @survey = Survey.new
+    @question = @survey.questions.build
+    @option = @question.options.build 
+  end
+
+  def show
+    @survey = Survey.find(params[:id])
+  end
+
+  def create
+    @survey = Survey.new(survey_params)
+    # @survey.questions.first.user_id = current_user.user_id
+    if @survey.save
```

Comment:
do not save it in controller add a method in model and pass it everything required